.github/workflows/ci.yaml

```
@@ -0,0 +1,21 @@
+name: CI
+
+on:
+  pull_request:
+  push:
+    branches:
+      - main
+
+jobs:
+  build:
+    name: Build
+    runs-on: ubuntu-latest
+    timeout-minutes: 10
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-java@v4
+        with:
+          distribution: 'zulu'
+          java-version: '17'
+          architecture: ${{ inputs.architecture }}
+      - run: ./gradlew assemble
```

Would it make sense to use `build`. It assembles and also dose some further verification and checks.
```suggestion
      - run: ./gradlew build
```

---

Sounds good. 👍